plugins/source/aws/codegen/recipes/kms.go

```diff
@@ -53,8 +53,23 @@ func KMSResources() []*Resource {
 						IgnoreInTests: true,
 					},
 				}...),
-		},
-	}
+			Relations: []string{"Grants()"},
+		}, {
+			SubService:  "grants",
```

Comment:
`grants` or `key_grants`? Not sure.

Reply:
it's called `ListGrants` (https://docs.aws.amazon.com/kms/latest/APIReference/API_ListGrants.html) so let's go with `grants` 👍

Reply:
Agreed with `grants`

Reply:
Reopening this... Traditionally if a resource depends on another one we prefix it with the other's name, so `aws_kms_key_grants` makes more sense and helps communicate that it requires `aws_kms_keys`.

Reply:
I'm ok with `aws_kms_key_grants` given the explanation above.  It does provide more clarity.

Reply:
Went with `aws_kms_key_grants` in [`e4416a1` (#3441)](https://github.com/cloudquery/cloudquery/pull/3441/commits/e4416a1d1c468b2811582a9aaf45bd9a9aae4129)

Reply:
Sounds good 👍